### DISCUSSION TITLE
I dont have access to github copilot

### DISCUSSION THREAD

**Discussion by yartat** [community/community#47592] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hi,
I might have signed in to my co-pilot with a correct GitHub account. I tried authenticate device (Visual Studio 22), but I can't access to Copilot functionality. I tried re-installing my extension, signing out of GitHub in Visual Studio 22 and trying sign in. Nothing works, I can only see the sad output:
```
GitHub Copilot 1.73.2.0 (v1.73.2.0@4c9585ed8)
GitHub Copilot Agent v1.73.0
You don't have access to GitHub Copilot.
Please open https://github.com/login/device and enter the device activation code XXXX-XXXX that has been copied to the clipboard.
You don't have access to GitHub Copilot.
Please open https://github.com/login/device and enter the device activation code XXXX-XXXX that has been copied to the clipboard.
You don't have access to GitHub Copilot.
```

**Comment by SilasRodrigues19** [comment]
Hey @yartat 

Check if it is active on your account.

https://github.com/github-copilot/signup

**Reply by yartat** [reply]
Yes, I have active monthly subscription

**Comment by tuves** [comment, marked as answer]
Thank you for reporting this. The bug fix has now been deployed to production and affected users should be able to connect. 

Due to the recency of this incident, we are still investigating the contributing factors and will provide a more detailed update on cause and remediation in the February Availability Report, which we will publish the first Wednesday of March.

We sincerely apologize for any stress and/or frustration this has caused.

**Reply by yartat** [reply]
Thanks a lot! The problem was solved.

**Reply by diazemilianoariel** [reply]
hola , no puedo acceder al copilot , me arroja este triste error.

![Captura de pantalla 2023-03-25 104605](https://user-images.githubusercontent.com/100837288/227721323-c702e14f-1fe8-44b1-ab50-e7e2c2d43a76.png)

**Comment by mayagatt** [comment]
Hi, joining in the topic. 
I am on a student account and I also have to access to github co pilot.
Is there any solution to this?

Thanks

**Comment by danrleypereira** [comment]
I'm sharing what worked for me
In VSCode 2022:
1. open Tools -> Options
2. search 'copilot'
3. set 'Signin with Device Flow' to 'True'
4. now open 'github.com/login/device' (remember to open it on the browser logged in the correct account) and enter the code generated in the VSCode 2022.
5. Done

![image](https://github.com/community/community/assets/12025215/5549e072-a49b-41e3-b49a-4d56b05f1a4f)
